content/ja/telephone.md

```diff
@@ -90,27 +90,29 @@ menu:
   </div>
   <div class="card-reveal">
     <span class="card-title grey-text text-darken-4">PBXLについて <i class="mdi-navigation-close right red-text text-accent-3"></i></span>
-    <p>2006年に設立したPBXLは、第2種通信事業者免許取得し、日本でいちはやくクラウド型コミュニケーションサービスを提供するなど、通信ソリューションのパイオニアとして活躍してます。</p><p>PBXLは、高い技術を持つスタッフが丁寧且つ、きめ細やかな対応で導入から日々のオペレーションサポートを提供してます。生産性の高いパフォーマンス構築を実現すると伴に、オープンなコラボレーションとモバイルスタッフの力を活用してます。</p>
+    <p>PBXLは、日本でいちはやくクラウド型コミュニケーションサービスを提供するなど、通信ソリューションのパイオニアを自負しています。</p><p>通信の先進国であるアメリカとの強いコネクションを持ち、高い技術を持ったバイリンガルスタッフが常駐。
+最先端技術の提供だけでなく、ご要望に合わせた柔軟な対応と、「全てを任せられる」安心感をこころがけています。
+あなたのワークスタイルを、「もっと便利に、もっと自由に」。</p>
   </div>
 </div>
 
 弊社イソリアはPBXLとパートナーシップを持ち、クラウド型IP電話システムを高い技術と共にお客様に提供してます。
 
 <figure class="image-container">
-<img class="materialboxed right responsive-img" width="200" data-caption="eSolia Partner PBXL's Logo" alt="Graphic of PBXL Logo" src="/img/pbxl-logo.png" >
+<img class="materialboxed right responsive-img" width="200" data-caption="イソリアのパートナーのPBXLロゴ" alt="PBXLのロゴ画像" src="/img/pbxl-logo.png" >
 </figure>
 
 PBXLの“クラウド電話”サービスは、固定電話・iPhone・Androidより、「いつでも・どこでも」アクセス可能な最先端な電話システムの導入を実現してます。
 
-PBXLサービスで実現可能なこと:
+PBXLサービスで実現可能なソリューション:
 
-* PBX不要なフル機能を備えたターンキー方式電話システム
+* PBX端末不要なフル機能を備えたターンキー方式電話システム
 * スマートフォンインテグレーションまたは、BYOD(Bring Your Own Device)：私用端末の業務利用
-* 通信コスト削減：日本を含む 22 カ国への通話がかけ放題
+* 通信コスト削減：日本を含む 22 カ国への通話かけ放題オプションが利用可能
 * 事業継続計画(Business Continuity Plan)サポート
 * 転送機能：不在の場合は自動的にスマホや他のスタッフに転送
 * コールパーク機能：通話を保留にし別の電話機からそのコールに応答
-* 簡単に会議が開ける電話会議/３者通話
+* 簡単に会議が開ける電話会議/３者通話：通話中に新たに別の人を呼び出し3人で同時に会話可能
 * コールピックアップ(代理応答)：他の人の電話に代理で応答できる機能
 * メールでFAX受信：モバイル端末でFAXをメールで添付ファイルとして受信可能
 * 着信拒否(DND)：忙しくて電話に出られない時
```

Comment:
@HitomiTsuruya you explained the acronyms above, but not for the DND line. Please add the explanation tomorrow Friday.